mod/Jailbreak.Warden/Global/WardenPeaceBehaviour.cs

```diff
@@ -0,0 +1,80 @@
+﻿using CounterStrikeSharp.API;
+using CounterStrikeSharp.API.Core;
+using Jailbreak.Public.Behaviors;
+using Jailbreak.Public.Generic;
+using Jailbreak.Public.Mod.Plugin;
+using Jailbreak.Public.Mod.Warden;
+
+namespace Jailbreak.Warden.Global;
+
+public class WardenPeaceBehaviour : IPluginBehavior, IWardenPeaceService
+{
+
+    private readonly IWardenService _wardenService;
+    private readonly ICoroutines _coroutines;
+    private readonly IEventsService _eventsService;
+
+    private readonly float _muteTime = 10.0f;
+
+    public WardenPeaceBehaviour(IWardenService wardenService, ICoroutines coroutines, IEventsService eventsService)
+    {
+        _wardenService = wardenService;
+        _coroutines = coroutines;
+        _eventsService = eventsService;
+
+        Func<bool> firstWardenPeaceMuteCallback = () =>
+        {
+            PeaceMute(_muteTime, true);
+            return true;
+        };
+
+        _eventsService.RegisterEventListener("first_warden_event", firstWardenPeaceMuteCallback);
+    
+    }
+
+    public CCSPlayerController? GetWarden()
+    {
+        return _wardenService.Warden;
+    }
+
+    public void PeaceMute(float time, bool exemptWarden = false)
+    {
+
+        List<CCSPlayerController> alreadyMuted = new List<CCSPlayerController>();
```

Comment:
Why keep track of already muted players?

Reply:
Good point 😆